src/Nox.EntityFramework.SqlServer/SqlServerDatabaseProvider.cs

```diff
@@ -20,6 +22,36 @@ public SqlServerDatabaseProvider(IEnumerable<INoxTypeDatabaseConfigurator> confi
     {
     }
 
+    public override void ConfigureEntity(
```

Comment:
This implementation is the same as on the base class. Can we remove this if it's not required?

Reply:
Desired changes were applied